src/gt4py/next/iterator/transforms/symbol_ref_utils.py

```diff
@@ -13,44 +13,59 @@
 # SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Again for my sanity, maybe we can extract this change into a separate PR. Do we have tests for these utils?